src/test/java/com/microsoft/sqlserver/jdbc/tvp/TVPResultSetCursorTest.java

```diff
@@ -224,7 +224,18 @@ public void testInvalidStoredProcedureName() throws SQLException {
                 pstmt.execute();
             } catch (SQLException e) {
                 if (!e.getMessage().contains(TestResource.getResource("R_StoredProcedureNotFound"))) {
-                    fail(e.getMessage());
+                    // diff error msg if AE enabled
+                    int start = connectionString.toUpperCase().indexOf(Constants.COLUMN_ENCRYPTION_SETTING);
+                    if (-1 != start) {
+                        int end = connectionString.indexOf(";", start);
+                        String value = connectionString.substring(
+                                start + Constants.COLUMN_ENCRYPTION_SETTING.length() + 1,
+                                (-1 != end) ? end : connectionString.length());
+                        if (value.equalsIgnoreCase(Constants.ENABLED)
+                                && !e.getMessage().contains(TestResource.getResource("R_incorrectSyntaxP0"))) {
```

Comment:
Can you explain why this error message is thrown?

Reply:
that's the error returned by sql server when columns are encrypted